lazy-imports-mixin.html

```diff
@@ -0,0 +1,71 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+`lazy-imports`
+<link rel="import" href="./lazy-imports-import.html">
+Declarative lazy imports
+
+@demo demo/index.html
+-->
+<script>
+  (() => {
+    const __loadedImports = {};
```

Comment:
I wonder if we want to share this state between the behavior and mixin. Not a huge deal, since the browser dedupes, but if we're keeping the state for a reason, that reason applies to both behavior and mixin.